src/main/resources/com/google/api/codegen/ruby/main.snip

```diff
@@ -228,8 +222,10 @@
         scopes: ALL_SCOPES,
         client_config: {},
         timeout: DEFAULT_TIMEOUT,
-        app_name: "gax",
-        app_version: Google::Gax::VERSION
+        app_name: nil,
+        app_version: nil,
+        lib_name: "gapic",
+        lib_version: "0.1.0"
```

Comment:
Agreed that lib_version should not be hard-coded. The prior is hard coded, so lets just hard code for now.

Tracking as an issue: https://github.com/googleapis/toolkit/issues/1015.